inst/assumptions.m

```diff
@@ -111,10 +111,8 @@
       'd = x._assumptions.generator'
       'if d == {}:'
       '    astr = ""'
-      'elif all(d.values()):'  % all True so list them
-      '    astr = ", ".join(sorted([str(i) for i in d.keys()]))'
-      'else:'  % more complicated case, just the raw dict
-      '    astr = str(d)'
+      'else:'  % all True so list them
+      '    astr = ", ".join(sorted([(("" if d.values()[i] else "~") + str(d.keys()[i])) for i in range(len(d.keys()))]))'
```

Comment:
Hi, actually are two ways, i fix that concept too enabling it from sym, first the normal example:
```
octave:6> a=python_cmd('return Symbol("a", complex=True, real=False)')
a = (sym) a
octave:7> assumptions(a)
ans = 
{
  [1,1] = a: complex, ~real
}
```
and with the possible syntax:
```
octave:8> b=sym('b', 'complex', 'real', false)
b = (sym) b
octave:9> assumptions(b)
ans = 
{
  [1,1] = b: complex, ~real
}
```
i need check some things about it for the moment, this will be better with the pr https://github.com/cbm755/octsympy/pull/577 to normalize the input.